releases/v1.1.0-rc.0.toml

```diff
@@ -0,0 +1,52 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.0"
+
+pre_release = true
+
+preface = """\
+`containerd` provides a daemon for managing running containers.
+
+1.1 is the second major release for `containerd` with added support for CRI, the
```

Comment:
We should mention that the CRI version supported is `v1alpha2`, and containerd v1.1 only works with Kubernetes 1.10+.

Reply:
Mayben I will mention this down below in the support section

Reply:
There's also the table here https://github.com/containerd/containerd/pull/2225/files#diff-5cafcf46bfb947f8e58fa5c16d39018cR167
```
| CRI GRPC API | Unstable     |v1alpha2  | [api/](https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/apis/cri/runtime/v1alpha2) |
```